src/arkode/arkode_io.c

```diff
@@ -1998,16 +3066,22 @@ int arkPrintAllStats(void* arkode_mem, FILE* outfile, SUNOutputFormat fmt)
   /* Print relaxation stats */
   if (ark_mem->relax_enabled)
   {
-    retval = arkRelaxPrintAllStats(arkode_mem, outfile, fmt);
+    retval = arkRelaxPrintAllStats(ark_mem, outfile, fmt);
     if (retval != ARK_SUCCESS) { return (retval); }
   }
 
+  /* Print stepper stats (if provided) */
+  if (ark_mem->step_printallstats)
+  {
+    return (ark_mem->step_printallstats(ark_mem, outfile, fmt));
+  }
```

Comment:
Not here.  If we had a super trivial stepper that had no stats of its own, then it would still be "valid" to ask ARKODE to print all of its current statistics.